src/JobsContainer/JobsContainer.js

```diff
@@ -0,0 +1,9 @@
+import JobTable from "../JobTable/JobTable";
+
+const JobsContainer = (props) => {
+    return (
+        <JobTable apiFetch={props.apiFetch}/>
+    )
+}
+
+export default JobsContainer
```

Comment:
doesn't seem like there's any reason for this component to exist? with or without it the resulting HTML, CSS & JS generated for the website would be the same